src/components/Skill/Skill.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Box as MuiBox, BoxProps as MuiBoxProps } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+type TLevelRange = 0 | 1 | 2 | 3 | 4 | 5;
```

Comment:
The rest of the team didn't use `T` prefix for types. Maybe remove it so it will be consistent